src/renderer/webgl/shaders/SpinnerShader.mjs

```diff
@@ -60,12 +60,12 @@ export default class SpinnerShader extends DefaultShader {
     }
 
     set color(v) {
-        this._color = v;
+        this._color = new Float32Array(StageUtils.getRgbaComponentsNormalized(v));
```

Comment:
Those changes break the shader completely as values initialised in constructor for `_color` and `_backgroundColor` are not converted. I prefer those to be reverted as I can't see any reason for them.

Reply:
reverted